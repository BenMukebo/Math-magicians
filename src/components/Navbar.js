import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <header className="header">
    <nav className="navbar">
      <div className="header-title">
        <h1 className=""> Math magicians </h1>
      </div>
      <ul className="">
        <li className="nav-item">
          <Link className="nav-item-link active" to="/">
            Home
            <span className="">(current)</span>
          </Link>
        </li>
        <li className="nav-item">
          <Link className="nav-item-link" to="/calculator">
            Calculator
          </Link>
        </li>
        <li className="nav-item">
          <Link className="nav-item-link" to="/quote">
            Quotes
          </Link>
        </li>
      </ul>
    </nav>
  </header>
);
export default Navbar;
