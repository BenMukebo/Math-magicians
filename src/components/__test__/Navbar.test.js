/** @format */

import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../Navbar';
import { HashRouter } from 'react-router-dom';

it('matches navbar snapshot', () => {
  const tree = renderer.create(<HashRouter> <Navbar /></HashRouter>,).toJSON();
  expect(tree).toMatchSnapshot();
});
