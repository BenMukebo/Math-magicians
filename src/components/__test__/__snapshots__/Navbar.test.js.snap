// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches navbar snapshot 1`] = `
Array [
  " ",
  <header
    className="header"
  >
    <nav
      className="navbar flex-center"
    >
      <div
        className="header-title"
      >
        <h1>
          Math
          <span>
             magicians 
          </span>
        </h1>
      </div>
      <ul
        className="flex-center"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-item-link active"
            href="#/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-item-link"
            href="#/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-item-link"
            href="#/quote"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>,
]
`;
